tests/serverpod_test_server/test_e2e/auth_module_test.dart

```diff
@@ -172,4 +172,66 @@ void main() {
       expect(result, equals(true));
     });
   });
+
+  group('Given signed in user that wants to alter its user info', () {
+    setUp(() async {
+      var response = await client.authentication.authenticate(
+        'test@foo.bar',
+        'password',
+      );
+      assert(response.success, 'Failed to authenticate user');
+      await client.authenticationKeyManager
+          ?.put('${response.keyId}:${response.key}');
+      assert(
+          await client.modules.auth.status.isSignedIn(), 'Failed to sign in');
+    });
+
+    tearDown(() async {
+      await client.authenticationKeyManager?.remove();
+      await client.authentication.removeAllUsers();
+      await client.authentication.signOut();
+      assert(
+        await client.modules.auth.status.isSignedIn() == false,
+        'Still signed in after teardown',
+      );
+    });
+    test('when changing its user name then user name is updated.', () async {
```

Comment:
Suggestion: Add an empty line to differentate the different stages of the test.

The test is great and the reasons provided will be very appriciated from future devs.
But for readability sake it would be nice to add some empty line between logical units of the test.

For example a new line could be added between the first section where we fetch and modify a user and the last section where we refetch and validate the new name.

Reply:
Sure thing, will sort that out.